docs/02-for-app-authors/12-linter.md

```diff
@@ -419,6 +464,34 @@ Flatpak creates its own XDG config, cache and data directories under
 by the portal. Applications do not need to access the host's
 XDG directories or subpaths of them.
 
+### finish-args-wildcard-name-own-name
+
+**Exceptions allowed**: No
+
+Name is one of `freedesktop, gnome, kde`.
+
+The manifest has a `own-name` that is one of
+`org.freedesktop.*, org.gnome.*, org.kde.*`.
+
+Wildcard access to bus names in well known namespace is a security issue.
+
+Only exceptions to finish-args-wildcard-kde-own-name might be allowed
+in case of legacy tray usage.
+
+### finish-args-wildcard-name-talk-name
+
+**Exceptions allowed**: No
+
+Name is one of `freedesktop, gnome, kde`.
+
+The manifest has a `talk-name` that is one of
+`org.freedesktop.*, org.gnome.*, org.kde.*`.
+
+Wildcard access to bus names in well known namespace is a security issue.
+
+Only exceptions to finish-args-wildcard-name-talk-name might be allowed
```

Comment:
Please check this line especially, as I've split these up, I'm unsure, if it applies here too

Reply:
Nope that is not needed. Legacy tray requires own-name=org.kde.* specifically. We don't want to allow wildcard-kde-talk-name as that is a mistake.

Thanks.